### PR TITLE
add flask-restplus from pip

### PR DESCRIPTION
**Pull request by Minipada**
**Added package**: flask-restplus
**Pypi Link**: https://pypi.org/project/flask-restplus/
**Description**: Fully featured framework for fast, easy and documented API development with Flask
**Usage**: Making an API interacting with ROS. Flask is quite famous but for APIs, restplus gives some nice features such as integrated modelling, marshalling, swaggers and more. It is very easy to use if you already know some flask.

I added it as pip since there is no deb for it.

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1626,6 +1626,10 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python-flask-restplus-pip:
```
Comment (nuclearsandwich) [inline rosdep/python.yaml:1629]:
```suggestion
python3-flask-restplus-pip:
```

Per our [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1) this should be changed to `python3-flask-restplus-pip`.

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
A small change is recommended to conform to contribution guidelines.

**Comment by Minipada** [comment]
@nuclearsandwich I get your change, but what about old version. If I want to make it work on melodic for example. Will it install it properly too?

**Comment by nuclearsandwich** [comment]
> I get your change, but what about old version. If I want to make it work on melodic for example, which use python2. Will it install it properly too?

So the short answer is yes. The pip installer and pip itself does not make any distinction between python2 or python3 when fetching packages from pypi.

**Review by nuclearsandwich — DISMISSED** [review]

**Comment by nuclearsandwich** [comment]
@Minipada it's my fault, I always forget that when I suggest inline it will break the lexical sorting. The definition needs to be moved down with the rest of the python3 definitions and then this will be good to go.

**Comment by Minipada** [comment]
Thanks for the precision.
It's mine, I just saw the suggestion and went with it. I did PR in the past, so I should have checked too. It should be right now.

**Review by nuclearsandwich — APPROVED** [review]
Thanks for iterating!

**Review by jacobperron — APPROVED** [review]

**Pull request merged**